### ISSUE TITLE
Finalize Export messaging and behavior

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-client#549]
## Description

Now that we have end-to-end exporting of files in the client, we need to finalize messaging and behavior.

Resources:

* [zeplin screens](https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5d3e9d093de035355770bc26)
* [invision flowchart](https://projects.invisionapp.com/share/WKTCW3J7B34#/screens/377942793)
* [invision interaction model](https://projects.invisionapp.com/share/WKTCW3J7B34#/screens/377942956)
* [PR that integrated export functionality](https://github.com/freedomofpress/securedrop-client/pull/547)
* [Export demos (poor video quality)](https://drive.google.com/drive/folders/1rXAyVxYLzliehCDu8A9vEzNXtXn8qTfv)

**Comment by creviera** [comment]
Related to messaging we need to finalize: https://github.com/freedomofpress/securedrop-client/issues/556

**Comment by creviera** [comment]
Separated UI styling in its own Issue: https://github.com/freedomofpress/securedrop-client/issues/560 so that it is not blocked by changes in wording or behavior.

**Comment by creviera** [comment]
We also need to finalize messaging around: `MULTIPLE_PRINTERS_CONNECTED` and `MULTIPLE_DRIVES_CONNECTED`

**Comment by creviera** [comment]
Here is placeholder messaging for `MULTIPLE_DRIVES_CONNECTED`
![Screenshot from 2019-11-13 12-57-29](https://user-images.githubusercontent.com/4522213/68803826-51856d00-0615-11ea-8ffa-db973650db1c.png)

**Comment by ninavizz** [comment]
[Zeplin Draft](https://zpl.io/blArpK5), from **24 Nov** meeting to agree on messaging and basic flow.

Text may be copy/pasted from draft. Because there is not a specific GH Issue for content, alone, I will keep the canonical source-of-truth in copyediting for these windows, in Zeplin.

**Comment by ninavizz** [comment]
...also see [This UX Repo Issue](https://github.com/freedomofpress/securedrop-ux/issues/57) as canonical source-of-truth for allthings Export.

**Comment by ninavizz** [comment]
Yup... me to do!

**Comment by eloquence** [comment]
(This issue is quite dated an no longer useful as an organizing epic; we can track/implement iterative improvements independently. Closing this issue.)
